.github/workflows/ci-nymvpn-x-rust.yml

```diff
@@ -0,0 +1,106 @@
+name: ci-nymvpn-x-rust
+
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - ".github/workflows/ci-nymvpn-x-rust.yml"
+      - "nymvpn-x/src-tauri/**"
+      - "crates/nym-vpn-proto/**"
```

Comment:
Make sure to include all crates you're using here, or the whole crates directory to make it more robust